string command
float64[] args